wdn/templates_5.2/scss/components/_components.buttons.scss

```diff
@@ -4,76 +4,118 @@
 
 
 .unl .dcf-btn {
-  @include txt-xs;
-  @include ls-2;
+  @include txt-sm;
+  text-decoration: none;
+  transition: all $transition-duration-hover-off $transition-timing-fn-hover-off;
 }
 
 
-// CSS-TRICKS HIGH CONTRAST MODE
-// .unl .dcf-btn:focus {
-//   outline: none;
-// }
+.unl .dcf-btn:hover {
+  transition: all $transition-duration-hover-on $transition-timing-fn-hover-on;
+}
 
 
-.unl .dcf-btn-secondary:active,
-.unl .dcf-btn-tertiary:active {
-  color: $color-active-light-mode;
+.unl .dcf-btn:focus {
+  outline: 3px solid transparent; // https://sarahmhigley.com/writing/whcm-quick-tips/
```

Comment:
Need this comment?

Reply:
I'd prefer to keep this one to provide context.